sendgrid/sendgrid.py

```diff
@@ -58,3 +57,14 @@ def api_key(self, value):
     @property
     def impersonate_subuser(self):
         return self._impersonate_subuser
+
+
+class SendGridClient(SendGridAPIClient):

```

Comment:
Why did we subclass SendGridAPIClient here? It seems more natural for us to have just added the function send_mail to the base class, or just created a new class that wasn't a subclass of SendGridAPIClient